schema-engine/connectors/sql-schema-connector/src/flavour/sqlite/connection.rs

```diff
@@ -162,7 +163,7 @@ impl ToColumnTypeExt for &SqliteColumn {
             "TEXT" => ColumnType::Text,
             "REAL" => ColumnType::Double,
             "BLOB" => ColumnType::Bytes,
-            "INTEGER" => ColumnType::Int32,
+            "INTEGER" => ColumnType::Int64,
```

Comment:
opportunistic fix after https://github.com/prisma/prisma-engines/pull/4976 - Quaint returns bigints when we can't infer a decl_column_type. So we must infer it as an i64